archetype/engine-v2/src/test/resources/generator/context-values.mustache

```diff
@@ -0,0 +1,11 @@
+{{foo}}
```

Comment:
Can you experiment with combining mustache controls with context path ?
I.e. `{{^~test.var3}}` `{{/~test.var3}}` or `{{#..var3}}` `{{/..var3}}`